Building with gtkdocs fails

Building with gtk-doc option fails because of illegal character in COPYING. Presumably due to 1ea167e58fe1c240fd189feb62a8e78e3085e9f2

It doesn't really say:
```
Building documentation for colord-gtk
/run/build/colord-gtk/_flatpak_build/libcolord-gtk/cd-version.h:46: warning: Parameter description for CD_CHECK_VERSION::major is missing in source code comment block.
/run/build/colord-gtk/_flatpak_build/libcolord-gtk/cd-version.h:46: warning: Parameter description for CD_CHECK_VERSION::minor is missing in source code comment block.
/run/build/colord-gtk/_flatpak_build/libcolord-gtk/cd-version.h:46: warning: Parameter description for CD_CHECK_VERSION::micro is missing in source code comment block.
/run/build/colord-gtk/libcolord-gtk/cd-window.h:55: warning: Value description for CdWindowError::CD_WINDOW_ERROR_LAST is missing in source code comment block.

ERROR: Error in gtkdoc helper script:

ERROR: ['/usr/bin/gtkdoc-mkhtml', '--path=/run/build/colord-gtk/doc/api:/run/build/colord-gtk/_flatpak_build/doc/api', 'colord-gtk', '../colord-gtk-docs.xml'] failed with status 6
../colord-gtk-docs.xml:89: element include: XInclude error : ../../../COPYING contains invalid char

```

--- a/COPYING
+++ b/COPYING
@@ -55,7 +55,7 @@ modified by someone else and passed on, the recipients should know
 that what they have is not the original version, so that the original
 author's reputation will not be affected by problems that might be
 introduced by others.
-
+
   Finally, software patents pose a constant threat to the existence of
 any free program.  We wish to make sure that a company cannot
 effectively restrict the users of a free program by obtaining a
@@ -111,7 +111,7 @@ modification follow.  Pay close attention to the difference between a
 "work based on the library" and a "work that uses the library".  The
 former contains code derived from the library, whereas the latter must
 be combined with the library in order to run.
-
+
                   GNU LESSER GENERAL PUBLIC LICENSE
    TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
 
@@ -158,7 +158,7 @@ Library.
   You may charge a fee for the physical act of transferring a copy,
 and you may at your option offer warranty protection in exchange for a
 fee.
-
+
   2. You may modify your copy or copies of the Library or any portion
 of it, thus forming a work based on the Library, and copy and
 distribute such modifications or work under the terms of Section 1
@@ -216,7 +216,7 @@ instead of to this License.  (If a newer version than version 2 of the
 ordinary GNU General Public License has appeared, then you can specify
 that version instead if you wish.)  Do not make any other change in
 these notices.
-
+
   Once this change is made in a given copy, it is irreversible for
 that copy, so the ordinary GNU General Public License applies to all
 subsequent copies and derivative works made from that copy.
@@ -267,7 +267,7 @@ Library will still fall under Section 6.)
 distribute the object code for the work under the terms of Section 6.
 Any executables containing that work also fall under Section 6,
 whether or not they are linked directly with the Library itself.
-
+
   6. As an exception to the Sections above, you may also combine or
 link a "work that uses the Library" with the Library to produce a
 work containing portions of the Library, and distribute that work
@@ -329,7 +329,7 @@ restrictions of other proprietary libraries that do not normally
 accompany the operating system.  Such a contradiction means you cannot
 use both them and the Library together in an executable that you
 distribute.
-
+
   7. You may place library facilities that are a work based on the
 Library side-by-side in a single library together with other library
 facilities not covered by this License, and distribute such a combined
@@ -370,7 +370,7 @@ subject to these terms and conditions.  You may not impose any further
 restrictions on the recipients' exercise of the rights granted herein.
 You are not responsible for enforcing compliance by third parties with
 this License.
-
+
   11. If, as a consequence of a court judgment or allegation of patent
 infringement or for any other reason (not limited to patent issues),
 conditions are imposed on you (whether by court order, agreement or
@@ -422,7 +422,7 @@ conditions either of that version or of any later version published by
 the Free Software Foundation.  If the Library does not specify a
 license version number, you may choose any version ever published by
 the Free Software Foundation.
-
+
   14. If you wish to incorporate parts of the Library into other free
 programs whose distribution conditions are incompatible with these,
 write to the author to ask for permission.  For software which is
@@ -456,7 +456,7 @@ SUCH HOLDER OR OTHER PARTY HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH
 DAMAGES.
 
                      END OF TERMS AND CONDITIONS
-
+
            How to Apply These Terms to Your New Libraries
 
   If you develop a new library, and you want it to be of the greatest

